custom/icds_reports/tasks.py

```diff
@@ -14,12 +17,22 @@ def move_ucr_data_into_aggregation_tables():
         with connections[settings.ICDS_UCR_DATABASE_ALIAS].cursor() as cursor:
 
             path = os.path.join(os.path.dirname(__file__), 'sql_templates', 'update_locations_table.sql')
+            # This seems like it shouldn't be updated often. Should only happen
+            # if there's been a change in the table since yesterday?
```

Comment:
Not going to worry about this now. we can take a look at the logs now that we have timestamps later